apps/playground/src/app/components/ag-grid/data-manager-large/data-manager-large.component.ts

```diff
@@ -12,16 +14,24 @@ import {
 } from '@skyux/data-manager';
 
 import { GridOptions } from 'ag-grid-community';
+import { BehaviorSubject } from 'rxjs';
 
 import { CustomLinkComponent } from './custom-link/custom-link.component';
 import { columnDefinitions, data } from './data-set-large';
 
 @Component({
   selector: 'app-data-manager-large',
   templateUrl: './data-manager-large.component.html',
+  styleUrls: ['./data-manager-large.component.scss'],
   changeDetection: ChangeDetectionStrategy.OnPush,
+  encapsulation: ViewEncapsulation.None,
 })
 export class DataManagerLargeComponent implements OnInit {
+  @HostBinding('class.use-normal-dom-layout')
```

Comment:
do consumers have to add this info (the styling above, the host binding, turning off view encapsulation) in order to use top scroll?

Reply:
This was to support switching and to set heights when not using auto-height.